.github/workflows/get-previous-release/action.yml

```diff
@@ -5,6 +5,10 @@ inputs:
   CHANNEL:
     required: true
     description: "The channel to get the version to (ga, cloud, beta)."
+  POSTFIX:
+    required: false
+    default: "-rc"
```

Comment:
Same comments as before

Reply:
Removed, I've converted the `POSTFIX` into a env var